homeassistant/components/tuya/light.py

```diff
@@ -652,6 +652,7 @@ def color_mode(self) -> str:
         # else than "white".
         if (
             self.entity_description.color_mode
+            and self.entity_description.color_mode in self.device.function
```

Comment:
Interesting!

The platform currently approaches it as: "If it is set in the entity description, it must be present." Wrong 😄 

Nice catch!

However, I think instead of making an exception in this place, we should determine it properly in the constructor (like we do for all other DPCodes as well).